packages/state-manager/src/sagas/publicChannels/deleteChannel/deleteChannel.saga.ts

```diff
@@ -24,12 +24,10 @@ export function* deleteChannelSaga(
     return
   }
 
-  if (currentChannel === undefined) {
-    return
-  }
-
-  if (currentChannel.disabled) {
-    return
+  if (currentChannel) {
+    if (currentChannel.disabled) {
+      return
+    }
```

Comment:
you can replace this nested if with nice oneliner if (currentChannel?.disabled) return